### PR TITLE
Stupid chud midround syndie "visitor" shuttle

### PR DESCRIPTION
**Pull request by starch70**
## About the PR
Adds stupid chud midround (>15 minute) shuttle with 2 syndie assault/saboteur borg spawners (it chooses randomly) and a spawner for a syndie team leader (i dont actually know where its used)

## Why / Balance
evil gang

## Media
image

## Requirements

- [X] I have read and am following the [Pull Request and Changelog Guidelines](https://docs.spacestation14.com/en/general-development/codebase-info/pull-request-guidelines.html).
- [X] I have added media to this PR or it does not require an ingame showcase.

## Breaking changes

**Changelog**
:cl:
- add: added a funny haha midround syndie borg shuttle :godo:

## Summary by CodeRabbit

- **New Features**
	- Introduced a new shuttle event configuration for the "Syndicate Striker," enhancing gameplay with new entities and interactions.
	- Added a new entity, "UnknownShuttleStriker," which includes specific event and loading rules.
	- Included a preloaded grid for the "SyndieStriker," expanding shuttle event options.

- **Bug Fixes**
	- Updated weight property for the "UnknownShuttleFrontierProspector" to improve game balance.

### CODE REVIEW

File: Resources/Prototypes/_Goobstation/GameRules/unknown_shuttles.yml [hunk 1]
```diff
@@ -3,6 +3,18 @@
...
   - type: LoadMapRule
     preloadedGrid: FrontierProspector
+
+
+- type: entity
+  id: UnknownShuttleStriker
+  parent: BaseUnknownShuttleRule
+  components:
+  - type: StationEvent
+    startAnnouncement: null #why would i do this
```
Comment (Piras314) [inline Resources/Prototypes/_Goobstation/GameRules/unknown_shuttles.yml:16]: does null even work in yaml? also make sure you've tested this with `addgamerule` and not just manually spawning in the shuttle

Comment (starch70) [reply Resources/Prototypes/_Goobstation/GameRules/unknown_shuttles.yml:16]: also i just based my shit off the preexisting and denied borg invasion midround shittle and that one did have the stupid "null" code

**Comment by JoeHammad1844** [comment]
- At least one assault both should be guaranteed, two saboteurs would suck!

**Comment by starch70** [comment]
> At least one assault both should be guaranteed, two saboteurs would suck

true but the humanoid spawner already gets like 225 TC + a extra like 150 tc of basic gear for free so i think its fine

also i cant make it garunteed assault borg spawner and gambling is way funnier

**Comment by Aidenkrz** [comment]
this fucking sucks

**Pull request closed without merging**